### PR TITLE
Show the activity and create the overlay view for the performance. 

### PR DESCRIPTION
**Pull request by Sunwoo-Nam**
When the activity of the renderer process is in the background,
loading time and performance are degraded.
So, open the activity and modify the user to select the visibility.

### CODE REVIEW

**Comment by Sunwoo-Nam** [comment]
This patch is not our goal, but this can be one of a solution to the performance issue.

**Comment by Sunwoo-Nam** [comment]
Create the transparent overlay view and it places always on top.
And show the welcome page to describe Castanets.
Users can see if Castanets is running by the notification.

**Review by joonghunpark — APPROVED** [review]
LGTM. But please wait for someone else to review.

File: chrome/android/java/src/org/chromium/chrome/browser/AlwaysOnTopService.java [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+import org.chromium.base.Log;
+import org.chromium.chrome.R;
+import org.chromium.chrome.browser.firstrun.FirstRunActivity;
+
+public class AlwaysOnTopService extends Service {
+    private static final String TAG = "AlwaysOnTopService";
+    private static final int NOTIFICATION = R.string.app_name;
+    private static final String APP_NAME = "CASTANETS";
+
+    private ImageView imgView;
```
Comment (is46-kim) [inline chrome/android/java/src/org/chromium/chrome/browser/AlwaysOnTopService.java]: naming style is inconsistent. this variable is class member.

File: chrome/android/java/src/org/chromium/chrome/browser/AlwaysOnTopService.java [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+    private NotificationManager mNotificationManager;
+
+    @Override
+    public IBinder onBind(Intent arg0) { return null; }
+
+    @Override
+    public void onCreate() {
+        super.onCreate();
+
+        WindowManager mWindowManager = (WindowManager) getSystemService(WINDOW_SERVICE);
```
Comment (is46-kim) [inline chrome/android/java/src/org/chromium/chrome/browser/AlwaysOnTopService.java]: naming style is inconsistent. this variable is local variable.

File: chrome/android/java/res_chromium/layout/castanets_welcome.xml [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+                    android:gravity="center_horizontal"
+                    android:text="is running..."
+                    android:textColor="#AAAAAA"
+                    android:textSize="20sp" />
+                <TextView
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:layout_marginTop="30sp"
+                    android:gravity="center_horizontal"
+                    android:text="It works event if you hide this page."
```
Comment (suyambulingamrm) [inline chrome/android/java/res_chromium/layout/castanets_welcome.xml]: typo. ? 'event' -> 'even'

**Review by suyambulingamrm — COMMENTED** [review]
verified the behavior.
minor comment.

File: chrome/android/java/res_chromium/layout/castanets_welcome.xml (hunk 3 above)
Comment (Sunwoo-Nam) [reply chrome/android/java/res_chromium/layout/castanets_welcome.xml]: Done. Thanks.

File: chrome/android/java/src/org/chromium/chrome/browser/AlwaysOnTopService.java (hunk 2 above)
Comment (Sunwoo-Nam) [reply chrome/android/java/src/org/chromium/chrome/browser/AlwaysOnTopService.java]: Done. Thanks.

File: chrome/android/java/src/org/chromium/chrome/browser/AlwaysOnTopService.java (hunk 1 above)
Comment (Sunwoo-Nam) [reply chrome/android/java/src/org/chromium/chrome/browser/AlwaysOnTopService.java]: Done. Thanks.

**Pull request merged**